junction/src/main/java/org/killbill/billing/junction/plumbing/billing/DefaultInternalBillingApi.java

```diff
@@ -93,7 +95,8 @@ public BillingEventSet getBillingEventsForAccountAndUpdateAccountBCD(final UUID
         final Catalog currentCatalog =  catalogInternalApi.getFullCatalog(true, true, context);
 
         // Check to see if billing is off for the account
-        final List<Tag> accountTags = tagApi.getTags(accountId, ObjectType.ACCOUNT, context);
+        final List<Tag> tags = tagApi.getTagsForAccountId(false, context);
```

Comment:
We should rename the variable `tags` here and below (in `addBillingEventsForBundles`) to avoid any confusion. Maybe `tagsForAccount`?